pkg/datamodeldiagram/datamodel.go

```diff
@@ -86,7 +90,7 @@ func GenerateDataModelsWithPureModule(datagenParams *cmdutils.CmdContextParamDat
 }
 
 func GenerateDataModel(pclass cmdutils.ClassLabeler, outmap map[string]string, mod *sysl.Module,
-	stmts []*sysl.Statement, title, project, outDir string) {
+	stmts []*sysl.Statement, title, project, outDir string, epName bool) {
```

Comment:
Tiny bit confused here, because epName is a bool, should it be renamed to epNameMode or epNameFlag?